### DISCUSSION TITLE
UDIM Favorites in side menu

### DISCUSSION THREAD

**Discussion by jtsage** [StjerneIdioten/I3D-Blender-Addon#154] [Ideas & Feature Requests]
Not sure where would be the best - but I'd love if I could set UDIM favorites to appear in either the 'N' menu, or maybe the tools menu like the "udim material picker" (or something named an awful lot like that) did.  My workflow is usually model, hand unwrap, pack, then udim - and when I have to wade through a complex part grabbing all the galvo or grade 8 out, it'd be awesome if I could just click 1 button rather than CTRL+U, select the picker, select the tile.

**Comment by StjerneIdioten** [comment]
I would somehow have to save your selected favourites to your user preferences and show them somewhere for something like that to work, which is probably entirely possible. In the mean time there is actually already a solution built into Blender which can ease your workload a bit, although due to the way I have structured the UI and Blender's lack of customisability it isn't the prettiest.

Blender has a Quick Favourites where you can right click and add any operator to it and then you can access the Quick Favourites menu by pressing `q` and get a list of your favourites. So you can right click any of the `select` buttons in the UDIM Picker and have that material available to you through the Quick Favourites menu, but they are unfortunately all named "select" in the menu because Blender just pulls the label from the button 🤦🏻‍♂️ 

![image](https://user-images.githubusercontent.com/11218511/172438410-f338ea95-87e1-495b-9bfb-1cc0d9c211d3.png)
![image](https://user-images.githubusercontent.com/11218511/172438763-4b71b477-ad90-42d4-be9a-6df7360040f7.png)

While it wont entirely give you the solution you want, it might make it a bit easier to work with rather than having to do `ctrl-u` then selecting the UDIM Picker and then selecting a material over and over again. In this way you just press `q` followed by either picking from the list or making use of Blender's automatic hotkey thing, where you can press the underligned letter in the operator name. So it can be as quick as `q` followed by `s`. `e` or `l` to select between three different materials.

I hope this helps a little bit :-)
